roles/etcd/handlers/main.yml

```diff
@@ -10,19 +10,19 @@
   command: systemctl daemon-reload
   when: ansible_service_mgr == "systemd"
 
+- name: reload etcd
```

Comment:
good point, according to docs http://docs.ansible.com/ansible/playbooks_intro.html: Notify handlers are always run in the same order they are defined, not in the order listed in the notify-statement. 
And this is very confusing... We must inspect another handlers as well.